src/components/Search/Input/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { AutocompleteRenderInputParams } from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import './style.scss';
+
+function Input(props: AutocompleteRenderInputParams) {
+    return (
```

Comment:
Maybe it would be better to name function using a verb? ( for example renderInput )